### DISCUSSION TITLE
errors occur during run lammps

### DISCUSSION THREAD

**Discussion by xn-huang** [deepmodeling/deepmd-kit#1331] [Q&A]
Hello, I trained a dp potential using the latest gpu version of deepmd-kit. Then I ran the lammps in the latest GPU version of deepmd-kit using our dp potential. I know the lammps is the CPU version, so I only used CPU to run it.

Errors occur as follows:
Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2021-12-03 16:50:24.049024: W tensorflow/stream_executor/cuda/cuda_driver.cc:312] failed call to cuInit: UNKNOWN ERROR (303)
2021-12-03 16:50:24.049534: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (swarm01): /proc/driver/nvidia/version does not exist

But the MD simulation ran successfully and it was slow. If should I ignore the errors or how can I fix them?
Thanks.

**Comment by njzjz** [comment]
TensorFlow could not find your CUDA driver (`libcuda.so.1` and `/proc/driver/nvidia/version`) so it would fall back to CPU.

**Reply by xn-huang** [reply]
Thanks for your reply. But why does it find CUDA driver since the lammps is the CPU version? So if should I ignore the errors?
Thanks.

**Reply by njzjz** [reply]
You were likely to use the GPU version. It will fallback to CPU if the GPU is not available.

**Reply by xn-huang** [reply]
Ok. Thanks so much.
